modules/sigprofiler/1.0/sigprofiler.smk

```diff
@@ -89,22 +88,17 @@ max_sigs = {
     "DBS2976" : 20
 }
 
-##### RULES #####
```

Comment:
We really shouldn't be hard-coding references into a module like this. Can't this be set in the config?